configuration/messageproperties/messages_encountertypes_en.properties

```
@@ -65,7 +65,8 @@ ui.i18n.EncounterType.name.0f070640-279e-4ec0-9e6c-6ef1f6567030=Exposed Infant F
 ui.i18n.EncounterType.name.aa42cc6c-b9ee-4850-926c-dda4bb14d890=Tuberculosis Intake
 # mexicoClinic EncounterTypes
 ui.i18n.EncounterType.name.aa61d509-6e76-4036-a65d-7813c0c3b752=Consult
-ui.i18n.EncounterType.name.0d16a7c9-07fb-43f6-8984-dd7787f26a5a=Clinical History
+ui.i18n.EncounterType.name.0d16a7c9-07fb-43f6-8984-dd7787f26a5a=Historia clinica
+ui.i18n.EncounterType.name.40d024d9-c7ec-4d2a-a862-68cb87543c4b=Ultrasonido
```

Este archivo es por ingles. Debes agregar estas lineas a `messages_encountertypes_es.properties`.

```suggestion
ui.i18n.EncounterType.name.0d16a7c9-07fb-43f6-8984-dd7787f26a5a=Clinical History
ui.i18n.EncounterType.name.40d024d9-c7ec-4d2a-a862-68cb87543c4b=Ultrasound
```

---

Listo !!! ya hice las dos correciones en config-ces y aqui, hay algo mas que me haga falta ? aun no veo la nueva pestaña